Topic0/practiceTopic0/Practice4/ConcreteBuilder1.java

```diff
@@ -0,0 +1,32 @@
+package practiceTopic0.Practice4;
+
+public class ConcreteBuilder1 implements BuilderInterface{
+	
+	private Connection connection;
+	
+	public ConcreteBuilder1() {
+			connection = new Connection();
+	}
+	
+	@Override
+	public void buildUser() {
```

Comment:
I don't think this name is correct, because you are not building users, or passwords. You are building a connection and these parameters are part of what the connection needs to be built.
I would name these methods as:
withUser, withPassword, withDb and buildConnection. In this case with is more common when you create a fluent API (https://en.wikipedia.org/wiki/Fluent_interface) 
or just setUser, setPassword, setDb and buildConnection.
I would also name the class ConnectionBuilder since that is what it does.